constraint/constraint_pipeline.py

```diff
@@ -0,0 +1,25 @@
+import hail as hl
```

Comment:
Should we keep the pipeline code in gnomad-constraint or gnomad_qc? The code here will decide the version of context, mutation, genome and exome site table.